velox/functions/prestosql/window/tests/LeadLagTest.cpp

```diff
@@ -184,6 +184,24 @@ TEST_P(LeadLagTest, ignoreNullsInt64Offset) {
   assertResults(fn(fmt::format("c0, {} IGNORE NULLS", largeOffset)));
 }
 
+TEST_P(LeadLagTest, ignoreNullsZeroOffset) {
+  auto data = makeRowVector({
+      // Values.
+      makeNullableFlatVector<int32_t>({1, std::nullopt, 2, std::nullopt, std::nullopt}),
+      // Offsets.
+      makeFlatVector<int64_t>({0, 0, 0, 0, 0}),
+  });
+
```

Comment:
You need `createDuckDbTable({data});` here before assertResults because the SQL query run in assertResults will be on this table.

Reply:
thank you @kagamiori for the great reminder!